autofit/non_linear/abstract_search.py

```diff
@@ -14,7 +14,12 @@
 
 from autofit import exc

```

Comment:
BLACK !? :O

Reply:
Yeah they use it at my client so I got used to having it as a filewatcher which is convenient. Also I figured out how to make it format how I want